client/my-sites/people/delete-user/index.jsx

```diff
@@ -18,9 +18,9 @@ const Card = require( 'components/card' ),
 	FormButtonsBar = require( 'components/forms/form-buttons-bar' ),
 	AuthorSelector = require( 'blocks/author-selector' ),
 	UsersActions = require( 'lib/users/actions' ),
-	Gravatar = require( 'components/gravatar' ),
 	accept = require( 'lib/accept' ),
 	analytics = require( 'lib/analytics' );
```

Comment:
I tried it out with a new line, and the extra space makes it seem like `<Gravatar />` is not an internal dependency, so I guess it's a bit strange either way. It looks ok to me without the extra line since the `import` keyword is in a new block-ish, so thinking of leaving as is.